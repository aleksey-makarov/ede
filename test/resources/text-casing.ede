{
  "single": "Lorem ipsum-Phrase",
  "number": 1.0
}
---
{{ single | lowerHead }}
{{ single | upperHead }}
{{ single | toLower }}
{{ single | toUpper }}
{{ single | toTitle }}
{{ single | toCamel }}
{{ single | toPascal }}
{{ single | toSnake }}
{{ single | toSpinal }}
{{ single | toTrain }}
{{ number | toOrdinal }}
{{ single | toLower | upperHead }}
{% if (single | toLower) == "lorem ipsum-phrase" %}
single | lower == "lorem ipsum"
{% endif %}
