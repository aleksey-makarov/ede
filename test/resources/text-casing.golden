lorem ipsum-Phrase
Lorem ipsum-Phrase
lorem ipsum-phrase
LOREM IPSUM-PHRASE
Lorem Ipsum Phrase
loremIpsumPhrase
LoremIpsumPhrase
lorem_ipsum_phrase
lorem-ipsum-phrase
Lorem-Ipsum-Phrase
1st
Lorem ipsum-phrase
single | lower == "lorem ipsum"
